util/dvsim/FlowCfg.py

```diff
@@ -15,6 +15,9 @@
 from Deploy import Deploy
 from utils import VERBOSE, md_results_to_html, parse_hjson, subst_wildcards
 
+# A set of fields that can be overridden on the command line.
+_CMDLINE_FIELDS = {'tool'}
```

Comment:
All command line args should take precedence over what is set in the hjson files. This IMO should not be needed.

Reply:
On second thoughts, I am ok keeping this in for now. Will figure out later what needs to be done here.